src/components/Roadmap.js

```diff
@@ -96,9 +108,10 @@ const Roadmap = () => {
           const implemented = issues
             .filter(
               (issue) =>
-                issue.state === "closed" &&
+                (issue.state === "closed" &&
                 "allcontributors[bot]" !== issue.user.login &&
-                !!issue.pull_request
+                !!issue.pull_request && checkifMerge(issue.pull_request.toString())===true)
```

Comment:
Similarly here... don't need the `===true` since `checkifMerge()` returns a boolean.

Also just noticing, mind adjusting the name to something like `isMerged` since it's a boolean return?

Reply:
ok I will change the naming convention